src/lang/qbittorrent_zh_Hant.ts

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.0" language="zh_TW">
+<TS version="2.0" language="zh-Hant">
```

Comment:
I am not sure. I temporarily created the Chinese Traditional Locale on Transifex and downloaded the file. This is what it had...
However, I don't know how I can actually test that qt picks up this as Chinese Traditional and doesn't expect "zh_Hant".